src/frame.js

```diff
@@ -0,0 +1,30 @@
+var Frame = function() {
+  this.firstBowl = null;
+  this.secondBowl = 0;
+  this.result;
+  this.spare = false;
```

Comment:
with Booleans, use `is` or `has` prefixes to improve readability - rather like `?` methods in ruby.  So `isSpare` and `isStrike`.